clj/resources/static_content/js/dashboard.js

```diff
@@ -169,6 +169,18 @@ jQuery( function() { ( function( $$, $, undefined ) {
 
     // NOTE: Fix for Firefox: https://github.com/slipstream/SlipStreamUI/issues/443
     $(".ss-usage-gauge.ss-usage-gauge-drawn [filter='url(#inner-shadow)']")
-        .each(function(){ $(this).removeAttr("filter") });
+        .each(function(){ $(this).removeAttr("filter"); });
+
+    if ( $$.util.meta.isSuperUserLoggegIn() ) {
```

Comment:
Check that `isSuperUserLoggegIn` is the correct name.
If, yes, fix the typo. (should be `isSuperUserLoggedIn`)